librz/include/rz_bin.h

```diff
@@ -1005,6 +1010,8 @@ RZ_API RZ_OWN RzList *rz_bin_file_compute_hashes(RzBin *bin, RzBinFile *bf, ut64
 RZ_API RzList *rz_bin_file_set_hashes(RzBin *bin, RzList *new_hashes);
 RZ_API RzBinPlugin *rz_bin_file_cur_plugin(RzBinFile *binfile);
 RZ_API void rz_bin_file_hash_free(RzBinFileHash *fhash);
+RZ_API RZ_OWN RzBinGoInfo *rz_bin_file_find_golang_info(RZ_NONNULL RzBinFile *binfile);
+RZ_API void rz_bin_file_golang_info_free(RZ_NULLABLE RzBinGoInfo *go_info);
```

Comment:
Not a big fan of the language-specific API but I guess we have no other choice for now.